api-client/src/endpoints/createPOIsEndpoint.ts

```diff
@@ -20,29 +20,32 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<PoiModel>> =>
     )
     .withMapper(
       (json: Array<JsonPoiType>): Array<PoiModel> =>
-        json.map(poi => {
-          return new PoiModel({
-            path: normalizePath(poi.path),
-            title: poi.title,
-            content: poi.content,
-            thumbnail: poi.thumbnail,
-            availableLanguages: mapAvailableLanguages(poi.available_languages),
-            excerpt: poi.excerpt,
-            location: new LocationModel({
-              id: poi.location.id,
-              name: poi.location.name,
-              address: poi.location.address,
-              town: poi.location.town,
-              state: poi.location.state,
-              postcode: poi.location.postcode,
-              region: poi.location.region,
-              country: poi.location.country,
-              latitude: poi.location.latitude,
-              longitude: poi.location.longitude
-            }),
-            lastUpdate: moment.tz(poi.modified_gmt, 'GMT'),
-            hash: poi.hash
-          })
-        })
+        json
+          .filter(poi => poi.location.id && poi.location.name && poi.location.latitude && poi.location.longitude)
```

Comment:
I think we still want to show locations without longitude and latitude in the Pois list

Reply:
Hmm not sure about that. I think every POI that is relevant should have a longitude/latitude.

Reply:
i'm with @klinzo the poi feature does not make any sense for me without displaying them on the map and have an address. i think with the cms plugin a poi that has an address also has long/lat